core/src/player.rs

```diff
@@ -2068,10 +2068,6 @@ impl Player {
         &mut self.storage
     }
 
-    pub fn destroy(self) -> Renderer {
```

Comment:
What was the point of this method? 🤔 Why does it rescue the renderer, of all things? I assume it wasn't used anymore...

Reply:
See https://github.com/ruffle-rs/ruffle/pull/1326. It was used in exporter for taking a screenshot.

Reply:
Ah, I see, thanks!

Reply:
Ah, and it got disused by https://github.com/ruffle-rs/ruffle/pull/7594.